terminus-ui/src/confirmation/confirmation-modal.component.html

```diff
@@ -1,21 +1,27 @@
-<div
-  fxLayout="row"
-  fxLayoutAlign="center space-between"
-  [fxLayoutGap]="gap"
+<div 
+  fxLayout="column"
 >
-  <ts-button
-    class="qa-confirmation-cancel"
-    theme="warn"
-    format="hollow"
-    (clickEvent)="confirm.next(false)"
+  <p *ngIf="explanationTxt" class="explanation">{{ explanationTxt }}</p>
+  <br>
```

Comment:
We should never use line breaks for layout purposes. Rather use CSS or the `ts-vertical-spacing` directive.